src/alto-ui/Actions/__tests__/Actions.test.js

```diff
@@ -0,0 +1,127 @@
+import React from 'react';
+import { mount } from 'enzyme';
+
+import Actions from '../Actions';
+import Dropdown from '../../Dropdown';
+import Group from '../../Group';
+
+jest.mock('../../Dropdown', () => {
+  return props => {
+    return (
+      <div className="Dropdown" onClick={() => props.onClick} onKeyDown={() => props.onClick} role="menu" tabIndex="0">
+        {props.items.map(action => {
+          return (
+            <button key={`key_${action.key}`} id={`key_${action.key}`} onClick={() => action.onClick || props.onClick()} />
+          )
+        })}
+      </div>
+    )
+  };
+});
+
+jest.mock('../../Group', () => {
+  return  () => {
+    return <div className="Group" />;
+  };
+});
+
+describe('Actions', () => {
+  const defaultProps = {
+    className: 'classTest',
+    id: 'id',
+    items: [
+      {
+        icon: null,
+        key: '1',
+        onClick: jest.fn(),
+        title: 'Edit',
+      },
+      {
+        icon: null,
+        key: '2',
+        onClick: jest.fn(),
+        title: 'Edit',
+      },
+    ],
+    max: 1,
+  };
+
+  const getWrapper = ({ ...props }) => mount(<Actions {...defaultProps} {...props} />);
```

Comment:
```suggestion
  const getWrapper = props  => mount(<Actions {...defaultProps} {...props} />);
```

Reply:
Refactored in "Refactor after code review" commit